VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/host-hearing-list/staff-member-hearing-list/staff-member-hearing-list.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { ProfileService } from 'src/app/services/api/profile.service';
+import { VideoWebService } from 'src/app/services/api/video-web.service';
+import { ConferenceForHostResponse } from 'src/app/services/clients/api-client';
+import { ErrorService } from 'src/app/services/error.service';
+import { EventsService } from 'src/app/services/events.service';
+import { HearingVenueFlagsService } from 'src/app/services/hearing-venue-flags.service';
+import { Logger } from 'src/app/services/logging/logger-base';
+import { ScreenHelper } from 'src/app/shared/screen-helper';
+import { HostHearingListComponent } from '../host-hearing-list.component-base';
+
+@Component({
+    selector: 'app-staff-member-hearing-list',
+    templateUrl: '../host-hearing-list.component.html',
+    styleUrls: ['../host-hearing-list.component.scss']
+})
+export class StaffMemberHearingListComponent extends HostHearingListComponent implements OnInit {
+    constructor(
+        private errorService: ErrorService,
+        protected videoWebService: VideoWebService,
+        protected router: Router,
+        protected profileService: ProfileService,
+        protected logger: Logger,
+        protected eventsService: EventsService,
+        protected screenHelper: ScreenHelper,
+        protected hearingVenueFlagsService: HearingVenueFlagsService
+    ) {
+        super(videoWebService, router, profileService, logger, eventsService, screenHelper, hearingVenueFlagsService);
+    }
+
+    retrieveHearingsForUser() {
+        this.logger.debug('[JudgeHearingList] - Updating hearing list');
+        this.conferencesSubscription.add(
+            this.videoWebService.getConferencesForStaffMember().subscribe({
+                next: (data: ConferenceForHostResponse[]) => {
+                    this.logger.debug('[JudgeHearingList] - Got updated list');
```

Comment:
Change prefix to [StaffMemberHearingList]. And set it to be a property that is added here rather than changing it here and line 45